dev/com.ibm.ws.security.authentication.builtin/resources/com/ibm/ws/security/authentication/internal/resources/AuthenticationMessages.nlsprops

```diff
@@ -83,3 +83,7 @@ JAAS_LOGIN_COLLECTIVE_PLUGIN_UNAVAILABLE.useraction=No action is required.
 JCACHE_AUTH_CACHE_CLEARED_ALL_ENTRIES=CWWKS1125I: The authentication cache cleared all entries in the {0} JCache cache. 
 JCACHE_AUTH_CACHE_CLEARED_ALL_ENTRIES.explanation=The authentication cache clears all entries in the backing JCache cache when security or user registry configuration is modified, or when the DeleteAuthCache MBean is called. 
 JCACHE_AUTH_CACHE_CLEARED_ALL_ENTRIES.useraction=No action is required.
+
+JCACHE_AUTH_CACHE_SERIALIZATION_FAILED=CWWKS1126W: Insertion of entry for the {0} key into the {1} JCache failed due to a serialization error. The entry will be inserted into the in-memory cache instead. This warning will not be emitted again. The exception is: {1}
+JCACHE_AUTH_CACHE_SERIALIZATION_FAILED.explanation=The contents of the entry could not be serialized so they could not be stored in the JCache cache.
```

Comment:
Change to remove the "could":

The entry contents were not serialized so they might not be stored in the JCache cache.

Reply:
Changed to:

`The entry contents were not serialized so they were not stored in the JCache cache.`